Library/Formula/vobcopy.rb

```diff
@@ -14,6 +11,11 @@ def install
                              "--with-lfs",
                              "--with-dvdread-libs=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/vobcopy -h 2>&1", 1)
```

Comment:
I could not get a working test based on the W6RZ DVD, either. Maybe this thing has just rotted too long and needs to be retired to the boneyard…